protocol/contracts/leaser/src/migrate.rs

```diff
@@ -51,7 +51,9 @@ where
     legacy_leases.and_then(|()| {
         customers
             .find_map(|maybe_customer| match maybe_customer {
-                Ok(customer) => msgs.migrate_or_be_next(customer, &migrate_msg),
+                Ok(customer) => {
+                    msgs.migrate_or_be_next(customer, &|| migrate_msg(MigrationKind::REGULAR))
```

Comment:
Taking a reference to the closure is most likely unnecessary and might also be caught by clippy in the CI.

Reply:
neither of them - the compiler complains if not provided, and the clippy is happy with it